src/Composer/ComposerLockReader.php

```diff
@@ -33,33 +38,50 @@ public function __construct(IntegratorConfig $config)
     public function getModuleVersions(): array
     {
         $composerLockData = $this->getProjectComposerLockData();
-        if (!isset($composerLockData['packages'])) {
-            return [];
-        }
-
         $packages = [];
 
-        /** @var array $packageData */
-        foreach ($composerLockData['packages'] as $packageData) {
-            if ($packageData['version'] === 'dev-master') {
+        foreach (['packages', 'packages-dev'] as $packagesKey) {
+            if (!isset($composerLockData[$packagesKey])) {
                 continue;
             }
-            [$org, $module] = explode('/', $packageData['name']);
-            $packageName = TextCaseHelper::dashToCamelCase($org) . '.' . TextCaseHelper::dashToCamelCase($module);
-            $packages[$packageName] = $packageData['version'];
-
-            if (strpos($packageData['version'], 'dev-') !== false) {
-                $versionFromExtra = $packageData['extra']['branch-alias']['dev-master'] ?? false;
-                if ($versionFromExtra) {
-                    $aliasedVersion = str_replace('x-dev', '0', $versionFromExtra);
-                    $packages[$packageName] = $aliasedVersion;
+            /** @var array $packageData */
+            foreach ($composerLockData[$packagesKey] as $packageData) {
+                if ($packageData['version'] === 'dev-master') {
```

Comment:
potentially we can skip here all branches that start with 'dev-', but it's ok for 'dev-master' first.